### ISSUE TITLE
`all-repos-list-repos`: add `--output-paths`

### ISSUE THREAD

**Issue by asottile** [asottile/all-repos#93]
The current output is a little clunky when trying to chain with xargs:

```console
$ all-repos-list-repos | tail -5
Yelp/docker-push-latest-if-changed
Yelp/requirements-tools
Yelp/venv-update
Yelp/wsgi-mod-rpaf
Yelp/yelp_cheetah
```

For instance:

```console
$ all-repos-list-repos | tail -5 | xargs --replace bash -c 'echo -n {}" " && git -C repos-asottile/{} rev-parse HEAD'
Yelp/docker-push-latest-if-changed 1d481bbbb30f1aa164bd965ba33ef61cbb11981c
Yelp/requirements-tools f94e0eb774bb4e37a58e98d5adbbe65a5520b8ef
Yelp/venv-update 02860446c983f8df10db221d66e22a7a912a550c
Yelp/wsgi-mod-rpaf d28a42fff8c77e5480bd6c166e4e53c2d308e81b
Yelp/yelp_cheetah 61ff00fd8944f19bc81b3f03801c322fc339ffeb
```

It would be nicer if there were this output:

```console
$ all-repos-list-repos --output-paths | tail -5
repos-asottile/Yelp/docker-push-latest-if-changed
repos-asottile/Yelp/requirements-tools
repos-asottile/Yelp/venv-update
repos-asottile/Yelp/wsgi-mod-rpaf
repos-asottile/Yelp/yelp_cheetah
```

**Comment by mitzkia** [comment]
Hi Anthony, 

I would like to give a possible fix for you request.
But before I started to maintain it I would like to have some questions to you (related to this topic):
1, As I understand, you would like to see the configured `output_dir` before each repos. Is it true?
2, Please consider to use the `--output-paths` cli argument, as it is already used with a different functionality. I would propose the `--relative-path` cli argument instead.
3, Would you like to use this (in what we agreed) cli argument in other `all-repos` functionality? (all-repos-find-files, all-repos-grep)

Thanks

**Comment by asottile** [comment]
1. yep, basically `os.path.join(config.output_dir, repo)`
2. Let's use `--output-paths` so it is consistent and not add a new flag
3. See 2.

thanks!

**Comment by mitzkia** [comment]
Realted PR: #95

**Comment by asottile** [comment]
> Realted PR: #95

btw I edited in `Resolves #95` into your PR description such that the issue will auto-close when I merged the PR -- thanks again for the patch :tada:
